dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheStoreService.java

```diff
@@ -83,13 +89,36 @@
      * @param props service properties
      */
     @Activate
-    @FFDCIgnore(CacheException.class)
+    @FFDCIgnore({ CacheException.class, URISyntaxException.class })
```

Comment:
Why ignore this FFDC if there is a TODO in the catch block indicating that we will later be logging an error for it anyway?

Reply:
I wasn't entirely sure what to do here in terms of logging the error/FFDC, thus the todo.  I'll remove it from the FFDCIgnore